cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/etl/transform/ScriptFilterTransform.java

```diff
@@ -98,6 +102,10 @@ private void init() {
     ScriptEngineManager manager = new ScriptEngineManager();
     engine = manager.getEngineByName("JavaScript");
 
+    // initialize metrics and logging
+    engine.put("metrics", metrics);
+    engine.put("LOG", logger);
```

Comment:
ok, i am just wondering how we should set validators.. should we namespace them with CDAP as well ? currently its set as separate global variables and referenced liked, ``Eg: coreValidator.maxLength(input.body, 10)``